recipes/menuinst/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.4.8" %}
+{% set sha256 = "053994cea32d71acf2846efc6dbd97b2fe5a4e98fa105ac36098d0a6d9c4ad9e" %}
+
+package:
+  name: menuinst
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ContinuumIO/menuinst/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - menuinst = menuinst.main:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pywin32
+
+test:
+  imports:
+    - menuinst
+
+  commands:
+    - menuinst --help
+
+about:
+  home: https://github.com/ContinuumIO/menuinst
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
+  summary: "cross platform install of menu items"
+
+extra:
+  recipe-maintainers:
+    - goanpeca
```

Comment:
Are you ok being a maintainer here, @goanpeca?

Reply:
Sure

Reply:
Thanks.